selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/FilterOrganizationTest.java

```diff
@@ -25,82 +27,81 @@
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteByName(organizationName);
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
   }
 
   @Test
   public void testOrganizationListFiler() {
-    navigationBar.waitNavigationBar();
     int organizationsCount = organizations.size();
 
-    navigationBar.clickOnMenu(NavigationBar.MenuItem.ORGANIZATIONS);
+    // Test that organization exist
+    navigationBar.clickOnMenu(ORGANIZATIONS);
     organizationListPage.waitForOrganizationsToolbar();
-    organizationListPage.clickAddOrganizationButton();
-
-    addOrganization.waitAddOrganization();
-    addOrganization.setOrganizationName(organizationName);
-    addOrganization.checkAddOrganizationButtonEnabled();
-    addOrganization.clickCreateOrganizationButton();
-    organizationPage.waitOrganizationTitle(organizationName);
+    organizationListPage.waitForOrganizationsList();
+    assertEquals(navigationBar.getMenuCounterValue(ORGANIZATIONS), organizationsCount);
+    assertEquals(organizationListPage.getOrganizationListItemCount(), organizationsCount);
+    assertTrue(organizationListPage.getValues(NAME).contains(ORGANIZATION_NAME));
 
-    assertEquals(
-        navigationBar.getMenuCounterValue(NavigationBar.MenuItem.ORGANIZATIONS),
-        String.valueOf(organizationsCount + 1));
-    navigationBar.clickOnMenu(NavigationBar.MenuItem.ORGANIZATIONS);
-    organizationListPage.waitForOrganizationsToolbar();
+    // Tests filter the organization by full organization name
+    organizationListPage.typeInSearchInput(ORGANIZATION_NAME);
     organizationListPage.waitForOrganizationsList();
-    assertEquals(organizationListPage.getOrganizationListItemCount(), organizationsCount + 1);
-    assertTrue(
-        organizationListPage
-            .getValues(OrganizationListPage.OrganizationListHeader.NAME)
-            .contains(organizationName));
+    assertTrue(organizationListPage.getValues(NAME).contains(ORGANIZATION_NAME));
+    assertEquals(organizationListPage.getOrganizationListItemCount(), 1);
 
-    // Tests search:
-    organizationListPage.typeInSearchInput(organizationName);
+    // Tests filter the organization by part of organization name
+    organizationListPage.clearSearchInput();
+    organizationListPage.typeInSearchInput(
+        ORGANIZATION_NAME.substring(ORGANIZATION_NAME.length() / 2));
+    organizationListPage.waitForOrganizationsList();
+    assertTrue(organizationListPage.getValues(NAME).contains(ORGANIZATION_NAME));
     assertEquals(organizationListPage.getOrganizationListItemCount(), 1);
-    organizationListPage.typeInSearchInput(organizationName + "test");
+
+    // Test filter the organization by wrong name
+    organizationListPage.clearSearchInput();
+    organizationListPage.typeInSearchInput(ORGANIZATION_NAME + "test");
```

Comment:
It's better to use obvious **"wrong_name"** string in search to avoid potential confusion.